tests/CommandTestBase.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from commands2 import Requirement
+from commands2.CommandScheduler import CommandScheduler
+
+
+class ValueHolderRequirement(Requirement):
+    value = 0
+
+    def set_value(self, value):
+        self.value = value
+
+
+class CommandTestBase(unittest.TestCase):
```

Comment:
Is there a reason you're using python unittest library? `pytest` is much less verbose and is preferred (and is what our other tests use).

Reply:
Nope, this is what IntelliJ gave me and I didn't know better to change.